build.gradle.kts

```diff
@@ -79,7 +79,7 @@ val unzipConfigurationSchema by tasks.registering(Copy::class) {
     val pathParts = path.split("/")
     path = pathParts.subList(1, pathParts.size).joinToString("/")
   })
-  into("$buildDir/semantic-conventions-${semanticConventionsVersion}/")
+  into(layout.buildDirectory.file("semantic-conventions-${semanticConventionsVersion}/"))
```

Comment:
I think this should use `dir` instead of `file`. Idk whether using `file` here breaks anything but the api seems to distinguish between regular files and directories.